docs/development/contribution-documentation/documentation-style-guide/README.md

```diff
@@ -1,10 +1,12 @@
----
+```
```

Comment:
One more issue. 

I'm sorry, I oversaw that the header markings have been changed in the markdown, too.
The yaml header needs to be enclosed by three dashes --- not ```

Reply:
hi @as-op 
okay, cool, i did that now, please check, thank you